src/panels/config/automation/trace/ha-automation-trace-logbook.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+import type { HomeAssistant } from "../../../../types";
+import type { LogbookEntry } from "../../../../data/logbook";
+import "../../../../components/trace/hat-logbook-note";
+import "../../../logbook/ha-logbook";
+
+@customElement("ha-automation-trace-logbook")
+export class HaAutomationTraceLogbook extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property({ attribute: false }) public logbookEntries!: LogbookEntry[];
+
+  protected render(): TemplateResult {
+    return html`
+      <ha-logbook
+        .hass=${this.hass}
+        .entries=${this.logbookEntries}
+      ></ha-logbook>
+      <hat-logbook-note></hat-logbook-note>
+    `;
+  }
+
+  static get styles(): CSSResult[] {
+    return [
+      css`
+        :host {
+          display: block;
+          padding: 16px;
```

Comment:
This means we now have extra padding on the logbook entries that already have padding?

Reply:
oh this should have been on the note. oops.